src/foundry/client/pixi/board.d.ts

```diff
@@ -578,41 +645,35 @@ declare global {
 
     /**
      * Handle left mouse-click events occurring on the Canvas.
-     * @see {@link MouseInteractionManager#_handleClickLeft}
```

Comment:
Were these links removed in Foundry? If so feel free to remove them.

Are these links not working? If so we can try to sort them out.

Reply:
Broken links; foundry uses `@see {MouseInteractionManager##handleClickLeft}`